bob-backend/src/pearl/disk_controller.rs

```diff
@@ -503,4 +503,16 @@ impl DiskController {
             group.close_unneeded_active_blobs(soft, hard).await;
         }
     }
+
+    pub(crate) async fn filter_memory_allocated(&self) -> usize {
+        let mut memory = 0;
+        for holder in self.groups.read().await.iter() {
```

Comment:
Maybe it's better to use `FuturesUnordered` here, so requests will be performed in parallel?

Reply:
And looks like it's `group`, not holder